config/locales/enumerators/pl.rb

```diff
@@ -0,0 +1,9 @@
+I18n.define_enumerator :pl do
+  def enumerate kind, list, options
+    conjunction = { :all => ' oraz ', :any => ' lub ', :one => ' albo ' }[kind]
+    if 2 >= list.length
+      return list.join conjunction
+    end
+    return list[0...-1].join(', ') + conjunction + list[-1]
+  end
+end
```

Comment:
I was thinking about organising it by language groups which use the same syntax, but was unable to find any information about it. Since I couldn't think of any other logical organisation I decided to just left it for now as separate implementations for each language.

If you have any idea where I could look for comparison of conjunctions in language groups, or can think of other ways to organise enumerator implementations, I would gladly look into it more.